.github/workflows/test.yaml

```diff
@@ -19,14 +20,9 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
     - name: Install project
-      run: pip install ".[all]"
+      run: pip install ".[test]"
     - name: Run unit tests
       env:
         FORCE_COLOR: 3
-      run: pytest -vvv |& tee unit_test_log${{ matrix.python-version }}.log
```

Comment:
No uploading of the logs? Wouldn't these be needed to debug test failures?

Reply:
I did have it that way, but for some reason Github Actions isn't able to know if the test fail or pass, so I removed it so that the CI can detect the test status.

Reply:
You may need to set pipefail so that you don't just get the exit code of `tee`, which is probably always 0. See: https://stackoverflow.com/questions/6871859/piping-command-output-to-tee-but-also-save-exit-code-of-command

Reply:
Omg! Thank you!